src/card.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  font-family: "Roboto", Helvetica, sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__information,
+.card__price,
+.card__rating,
+.card__button {
+  margin: 0 16px 16px;
+}
+
+.card__rating,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.card__information h2 {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__information p {
+  margin-top: 4px;
+  color: #616070;
+}
+
+.card__price-price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
```

Comment:
"what if the font size increases?" - i mean, that imho it woud be better if u set padding instead certain height, so if u increase button text size in future, then ur button will increase it`s size 2